src/core/reducers/errors.js

```diff
@@ -17,15 +17,19 @@ function getMessagesFromError(error) {
 
     Object.keys(error.response.data).forEach((key) => {
       const val = error.response.data[key];
-      val.forEach((msg) => {
-        if (key === 'non_field_errors') {
-          // Add generic messages for the API response.
-          apiMessages.push(msg);
-        } else {
-          // Add field specific error messages.
-          apiMessages.push(`${key}: ${msg}`);
-        }
-      });
+      if (Array.isArray(val)) {
+        val.forEach((msg) => {
+          if (key === 'non_field_errors') {
```

Comment:
If it's the only place it's used I guess it's fine, but if this is a special key from the API telling the front-end it's a generic error rather than one associated with a specific key (as it seems to be), storing that value in a constant seems our usual route, similar to using `ADDON_TYPE_THEME`

Maybe it's just because `'non_field_errors'` wasn't immediately obvious to me. A comment would help maybe.